tests/UI/pages/FO/home/newProducts.ts

```diff
@@ -0,0 +1,23 @@
+// Import pages
+import FOBasePage from '@pages/FO/FObasePage';
```

Comment:
Duplicate of `tests/UI/pages/FO/newProducts/index.ts` ?

Reply:
Done :+1: